core-web/libs/sdk/angular/project.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "sdk-angular",
+    "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+    "sourceRoot": "libs/sdk/angular/src",
+    "prefix": "lib",
+    "tags": [],
```

Comment:
Since this is a publishable library, i think it doesnt need to have tags
thoughts? cc: @fmontes @zJaaal @rjvelazco

Reply:
> Since this is a publishable library, i think it doesnt need to have tags thoughts? cc: @fmontes @zJaaal @rjvelazco

Yes, I know is a publishable lib, but this file is a project.json, could be nice group by tag all the sdk libs, like `"tags": ["type:sdk", "scope:client"]`.